src/cephadm/cephadm.py

```diff
@@ -2872,7 +2881,6 @@ def get_container(
         d_args.extend(sg.get_daemon_args())
 
     _update_container_args_for_podman(ctx, ident, container_args)
-    mounts = get_container_mounts(ctx, ident)
```

Comment:
Do you think which types need the container mounts is something worth being written down as part of a class variable or something or would it not be worth it? I'm sort of reminded of when there was a `uid_gid_daemon` or something type of daemon form, where there was a case for the distinction but we decided it wasn't worth the effort.

Reply:
discussed this in person, not worth doing